packages/loader/src/TextureLoader.ts

```
@@ -0,0 +1,204 @@
+
+class TextureContentRestorer extends ContentRestorer<Texture> {
+  constructor(
+    resource: Texture,
+    public url: string,
+    public requestConfig: RequestConfig
+  ) {
+    super(resource);
+  }
+
+  override restoreContent(): AssetPromise<Texture> {
+    return this.resource.engine.resourceManager
```

ESLint

[error] Insert `(⏎······`

(prettier/prettier)